Formula/pdns.rb

```diff
@@ -26,54 +27,79 @@ class Pdns < Formula
   deprecated_option "with-pgsql" => "with-postgresql"
 
   depends_on "pkg-config" => :build
+  depends_on "cmake" => :build
   depends_on "boost"
   depends_on "lua"
   depends_on "openssl"
   depends_on "sqlite"
   depends_on :postgresql => :optional
 
+  resource "pdns-brew-default-config" do
+    url "https://github.com/mprzybylski/pdns-brew-default-config/archive/0.2.tar.gz"
```

Comment:
Sorry, I'm not happy depending on another repository for this and adding an additional build-time dependency. We should use an upstream distributed configuration file that's suitably `inreplace`d to our needs or a very, very short configuration file embedded in the formula while we attempt to get upstream to distribute a sensible default file.